packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -367,35 +370,18 @@ export interface IFluidDataStoreContextEvents extends IEvent {
 }
 
 /**
- * Represents the context for the data store. It is used by the data store runtime to
- * get information and call functionality to the container.
+ * Represents the context for the data store like objects. It is used by the data store runtime to
+ * get information and call functionality to its parent.
```

Comment:
Maybe put some note here that this is temporary partitioning, and that long term we expect to have no differences between this interface and the next one?